vic/drivers/python/src/display_current_settings.c

```diff
@@ -395,11 +395,11 @@ display_current_settings(int mode)
     fprintf(LOG_DEST, "Input State File:\n");
     if (options.INIT_STATE) {
         fprintf(LOG_DEST, "INIT_STATE\t\tTRUE\t%s\n", filenames.init_state);
-        if (options.BINARY_STATE_FILE) {
-            fprintf(LOG_DEST, "BINARY_STATE_FILE\tTRUE\n");
+        if (options.STATE_FORMAT == ASCII) {
```

Comment:
In all the other drivers you test BINARY before ASCII. It does not really matter but for readability and consistency swap the `if` and `else if` statements